lab-adam/server.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const http = require('http');
+const url = require('url');
+const cowsay = require('cowsay');
+const parseBody = require('./lib/parse-body.js');
+const queryString = require('querystring');
+const PORT = process.env.PORT || 3000;
+
+http.createServer(function(req, res) {
+  req.url = url.parse(req.url);
+  req.url.query = queryString.parse(req.url.query);
+  console.log(req.url.query.text);
```

Comment:
You should remove console.logs that are used for testing/debugging prior to submission. If they are being used as server logging, that's fine!